src/components/print/PrintDirective.js

```diff
@@ -660,22 +660,25 @@
       // Transform layers to literal
       angular.forEach(layers, function(layer) {
         if (layer.visible) {
-          var attribution = layer.attribution;
-          if (attribution !== undefined &&
-              attributions.indexOf(attribution) == -1) {
-            attributions.push(attribution);
-          }
-          if (layer instanceof ol.layer.Group) {
-            var encs = $scope.encoders.layers['Group'].call(this,
-                layer, proj);
-            encLayers = encLayers.concat(encs);
-          } else {
-            var enc = encodeLayer(layer, proj);
-            if (enc && enc.layer) {
-              encLayers.push(enc.layer);
-              if (enc.legend) {
-                encLegends = encLegends || [];
-                encLegends.push(enc.legend);
+          if (!layer.timeEnabled ||
+               (layer.timeEnabled && layer.time != undefined)) {
```

Comment:
I think we could potentially have a problem here if we have a Layer.Group which is `timeEnabled = false ` and its childs are `timeEnabled=true` what do you think?